homeassistant/components/coinbase/diagnostics.py

```diff
@@ -0,0 +1,33 @@
+"""Diagnostics support for Coinbase."""
+
+from homeassistant.components.diagnostics import async_redact_data
+from homeassistant.config_entries import ConfigEntry
+from homeassistant.const import CONF_API_KEY, CONF_API_TOKEN, CONF_ID
+from homeassistant.core import HomeAssistant
+
+from . import CoinbaseData
+from .const import API_ACCOUNT_AMOUNT, API_RESOURCE_PATH, CONF_TITLE, DOMAIN
+
+TO_REDACT = {
+    API_ACCOUNT_AMOUNT,
+    API_RESOURCE_PATH,
+    CONF_API_KEY,
+    CONF_API_TOKEN,
+    CONF_ID,
+    CONF_TITLE,
+}
+
+
+async def async_get_config_entry_diagnostics(
+    hass: HomeAssistant, entry: ConfigEntry
+) -> dict:
+    """Return diagnostics for a config entry."""
+    instance: CoinbaseData = hass.data[DOMAIN][entry.entry_id]
+
+    return async_redact_data(
+        {
+            "entry": entry.as_dict(),
+            "data": instance.accounts,
```

Comment:
```suggestion
            "accounts": instance.accounts,
```
I would still use "accounts" here as that is what it is, but up to you :+1:

Reply:
I agree entirely. Had that before but figured I go for consistency with other integrations. Will change back now